app/routes.php

```diff
@@ -0,0 +1,33 @@
+<?php 
```

Comment:
Adapt your Yaseen's routes implementation with a separate routes.php in config directory.